tasks/main.yml

```diff
@@ -1,20 +1,57 @@
 ---
-- name: Install targetcli packages
+- name: Install targetcli packages on RedHat/CentOS/Fedora
   yum:
     name: 'targetcli'
     state: 'installed'
+  when: 'ansible_os_family == "RedHat"'
 
-- name: Start and enable targetcli service
+- name: Install targetcli packages on SUSE 15.x / openSUSE 15.x or higher
+  package:
+    name: 'python3-targetcli-fb'
+    state: 'installed'
+    disable_recommends: true
```

Comment:
does this (`disable_recommends`) attribute work with `package` module (https://docs.ansible.com/ansible/latest/modules/package_module.html)? Looking at documentation I see it only in `zypper` module (https://docs.ansible.com/ansible/latest/modules/zypper_module.html#zypper-module). Additionally in `zypper` module it defaults to `yes`/`true` which would mean that we don't need to specify it.

Reply:
You are of course right, sorry. This was a leftover of my testing yesterday.